webui/src/pages/repositories/repository/tags.jsx

```diff
@@ -0,0 +1,236 @@
+import React, { useMemo, useRef, useState } from "react";
+
+import {
+    TagIcon,
+    LinkIcon,
+    PackageIcon,
+    SearchIcon
+} from "@primer/octicons-react";
+import ButtonGroup from "react-bootstrap/ButtonGroup";
+import Button from "react-bootstrap/Button";
+import Card from "react-bootstrap/Card";
+import ListGroup from "react-bootstrap/ListGroup";
+
+import { tags } from "../../../lib/api";
+
+import {
+    ActionGroup,
+    ActionsBar, ClipboardButton,
+    Error, LinkButton,
+    Loading, RefreshButton
+} from "../../../lib/components/controls";
+import { RepositoryPageLayout } from "../../../lib/components/repository/layout";
+import { RefContextProvider, useRefs } from "../../../lib/hooks/repo";
+import { useAPIWithPagination } from "../../../lib/hooks/api";
+import { Paginator } from "../../../lib/components/pagination";
+import Modal from "react-bootstrap/Modal";
+import Form from "react-bootstrap/Form";
+import RefDropdown from "../../../lib/components/repository/refDropdown";
+import { Link } from "../../../lib/components/nav";
+import { useRouter } from "../../../lib/hooks/router";
+
+
+const TagWidget = ({ repo, tag }) => {
+
+    const buttonVariant = "outline-dark";
+
+    return (
+        <ListGroup.Item>
+            <div className="clearfix">
+                <div className="float-left">
+                    <h6>
+                        <Link href={{
+                            pathname: '/repositories/:repoId/objects',
+                            params: { repoId: repo.id },
+                            query: { ref: tag.commit_id }
+                        }}>
+                            {tag.id}
```

Comment:
Could we add tip when the user hover that is something like:
"Explore objects at the tag"